code/kg2/prompt_for_password_and_save_to_temp_file.py

```diff
@@ -16,10 +16,18 @@
 import getpass
 import os
 import tempfile
-
+os.sys.path.append("..") #make modules in the code/ directory accessible
+from RTXConfiguration import RTXConfiguration
 
 if __name__ == '__main__':
-    password = getpass.getpass("Please enter the password: ")
+    try:
+        rtxc = RTXConfiguration()
```

Comment:
due to the way RTXConfiguration works, I think you may need to add one more line before you access `rtxc.neo4j_password`:
```
rtxc.live = "KG2"
```
(that flips it into 'KG2 mode', which means the KG2 login info will be used. otherwise, it will use the KG1 login info by default.)